src/garage/torch/policies/continuous_mlp_policy.py

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
So this class as written isn't actually specific to MLPs. Please rename the file and class to `ContinuousNNPolicy` (or some other name that you feel is appropriate).

Reply:
to be consistent with the TF API, please also create a (very small) class which is actually `ContinuousMLPPolicy`. This is a really nice helper class though.

Reply:
I talked to @zhanpenghe yesterday, and we talked about making the this policy class module agnostic and adding a parameter that stores the policy type (deterministic or stochastic). If type is deterministic, then get_action simply returns the forward call. Otherwise, it samples from the distribution returned by the forward call. Does that sound okay to everyone?

Also, we talked about not following the TF implementation since torch does things differently, so do we still want a ContinuousMLPPolicy class in this case? Lmk what you guys think.

Reply:
I think we should have a `ContinuousMLPPolicy` helper class, but it can wait for a later PR.

Reply:
changing your type behavior based on constructor parameters is an antipattern, even if someone else did it in their algorithm. 

i think it's fine to wait for `ContinuousMLPPolicy` in a different PR, but in that case please update the commit message and PR description (and file an issue, assign it to yourself).